selectors.yml

```diff
@@ -3,18 +3,36 @@ selectors:
   31:
     selector: 8953668971247136127
     name: "bitcoin-testnet-rootstock"
+  41:
+    selector: 729797994450396300
+    name: telos-evm-testnet
+  45:
+    selector: 4340886533089894000
+    name: darwinia-testnet-pangoro
+  53:
+    selector: 8955032871639343000
+    name: coinex_smart_chain-testnet
+  81: 
+    selector: 6955638871347136141
+    name: "astar-testnet-shibuya" 
   97:
     selector: 13264668187771770619
     name: "binance_smart_chain-testnet"
+  111: 
+    selector: 572210378683744374 
+    name: "velas-testnet"
   195:
     selector: 2066098519157881736
     name: "ethereum-testnet-sepolia-xlayer-1"
-  196:
-    selector: 3016212468291539606
-    name: "ethereum-mainnet-xlayer-1"
```

Comment:
Maybe better not remove names which were already committed?

Reply:
i placed it in the mainnet section. It was placed in the testnet section 
https://github.com/smartcontractkit/chain-selectors/pull/40/files#diff-6724bdfc10769b3c4b08cdf2c057660d7ac98cedf4c3900fb5be959e034f84fbR209-R210